test/integration/plugin_test.go

```diff
@@ -62,8 +63,7 @@ func TestPluginEssentials(t *testing.T) {
 		"konghq.com/strip-path":     "true",
 	}, service)
 	require.NoError(t, clusters.DeployIngress(ctx, env.Cluster(), ns.Name, ingress))
-	err = cleanIngress(cleaner, ingress)
-	require.NoError(t, err)
+	addIngressToCleaner(cleaner, ingress)
```

Comment:
BTW: If we change `generators.NewIngressForServiceWithClusterVersion()`

to return `client.Object` instead of `runtime.Object` this won'd be necessary.